setup.py

```diff
@@ -90,6 +90,7 @@
 ]
 
 extras_require = setuptools_args['extras_require'] = {
+    'fast': ['fastjsonschema'],
```

Comment:
We should also add this to `test` so tests can use it without issue,

Reply:
Fixed!